Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -112,6 +116,14 @@ private void OnInteractUsingEvent(EntityUid uid, FlammableComponent flammable, I
             if (!isHotEvent.IsHot)
                 return;
 
+            // Only apply stacks to entities tagged with specific materials
+            // Otherwise  mobs would too easily be able to set each other on fire
+            // In future, support Wood, Fuel, Cloth, etc.
+            if (_tags.HasTag(uid, "Paper"))
+            {
+                flammable.FireStacks = 1;
```

Comment:
This will forcibly set the firestacks to 1 even if it's currently higher. Use `Math.Max(1, flammable.FireStacks)` instead.